tools/reviewer/web/src/styles/styles.scss

```diff
@@ -5,9 +5,6 @@
 @import 'global';
 @import 'reviewsPanel';
 @import 'login';
-
-@import 'review/singleReview';
-@import 'review/titlebar';
-
+@import 'review.component.scss';
```

Comment:
so we have moved scss to files within the component folders?

Reply:
Before that we had 3 files:
`reviewsPanel.scss`
`codeReview.scss`
`singleReview.scss`

How should developers know, which file describe review.component?

I renamed the `singleReview` to `review.component.scss`, to understand which scss file describes review.component and its children.

Reply:
I didn't ask that but thank you for bringing it up. I agree the names were not proper before but review.component.scss should be review.scss. There is no need for putting component with it. Also you need to specify extension while importing as visible in other imports.

I believe it was a folder of review and two files under the folder. I thought we had decided something else in the chat a couple days ago.